baybe/surrogates/gaussian_process.py

```diff
@@ -69,18 +69,12 @@ def _fit(self, searchspace: SearchSpace, train_x: Tensor, train_y: Tensor) -> No
         # ---------- GP prior selection ---------- #
         # TODO: temporary prior choices adapted from edbo, replace later on
 
-        mordred = searchspace.contains_mordred or searchspace.contains_rdkit
-        if mordred and train_x.shape[-1] < 50:
-            mordred = False
-
-        # low D priors
-        if train_x.shape[-1] < 5:
-            lengthscale_prior = [GammaPrior(1.2, 1.1), 0.2]
-            outputscale_prior = [GammaPrior(5.0, 0.5), 8.0]
-            noise_prior = [GammaPrior(1.05, 0.5), 0.1]
+        mordred = (searchspace.contains_mordred or searchspace.contains_rdkit) and (
+            train_x.shape[-1] >= 50
+        )
 
         # DFT optimized priors
-        elif mordred and train_x.shape[-1] < 100:
+        if mordred and train_x.shape[-1] < 100:
```

Comment:
the fix does a bit more than I would have expected, in meeting we identified the last block (else) as the issue which needs changing, why has all this other stuff been altered? Probably fine but regrading we have a last min fix ehre why take the risk?

Reply:
Ah ok, I thought we would remove all non-substance related settings. But I'm fine with whatever you consider most reasonable since I have no strong opinion and it's only a quickfix anyway. Shall I change the upper part back?

Reply:
yes due to keepin it robust Id prefer to revert to the 4block variant, and adjust the 4th block (to have the contents of the 1st block) OR structure the if else blocks in aw ay only 3 blocks are needed

Alternatively I think the problem is also temporarily fixed by changing `< 5` in the first block to something larger eg 20

Reply:
Ok, made some changes before I actually saw your message, where I now only changed to the botorch defaults in the fourth block. Is that OK or do you strictly prefer having some edbo content for that case as well?

Reply:
not hat also works I guess

Reply:
discussed in meeting --> resolve